include/llama.h

```diff
@@ -588,6 +588,12 @@ extern "C" {
             struct llama_model * model,
             const char * path_lora);
 
+    // Get the LoRA task name. Returns a blank string if not applicable
+    LLAMA_API const char * llama_adapter_lora_task_name(struct llama_adapter_lora * adapter);
+
+    // Get the required LoRA prompt prefix. Returns a blank string if not applicable
+    LLAMA_API const char * llama_adapter_lora_prompt_prefix(struct llama_adapter_lora * adapter);
```

Comment:
My opinion is that it is very hard to justify adding these APIs just for this model. However, more generic functions to read metadata from lora ggufs, analogous to the `llama_model_meta` functions, should be ok.